cypress/deployed-and-local/integration/irsSuperUser/irs-super-user-can-download-stin.cy.ts

```diff
@@ -0,0 +1,82 @@
+import { ROLES } from '../../../../shared/src/business/entities/EntityConstants';
+import { getCypressEnv } from '../../../helpers/env/cypressEnvironment';
+import { loginAsPetitioner } from '../../../helpers/authentication/login-as-helpers';
+import { petitionerCreatesElectronicCase } from '../../../helpers/fileAPetition/petitioner-creates-electronic-case';
+import { petitionsClerkQcsAndServesElectronicCase } from '../../../helpers/documentQC/petitions-clerk-qcs-and-serves-electronic-case';
+
+if (!Cypress.env('SMOKETESTS_LOCAL')) {
+  describe('irs superuser integration', () => {
+    const password = getCypressEnv().defaultAccountPass;
+    const userName = 'cypress_test_account_irs_super_user@example.com';
+
+    beforeEach(() => {
+      Cypress.session.clearCurrentSessionData();
+      cy.task('deleteAllIrsCypressTestAccounts');
+    });
+
+    after(() => {
+      cy.task('deleteAllIrsCypressTestAccounts');
+    });
+
+    it('should let an irs superuser view the reconciliation report and download a STIN', () => {
+      cy.task('createAccount', {
+        irsEnv: true,
+        password,
+        role: ROLES.irsSuperuser,
+        userName,
+      });
+
+      cy.task('getIrsBearerToken', { password, userName }).then(idToken => {
+        loginAsPetitioner();
+        petitionerCreatesElectronicCase().then(docketNumber => {
+          petitionsClerkQcsAndServesElectronicCase(docketNumber);
+          verifyReconciliationReportAndStinUrl(idToken as string);
+        });
+      });
+    });
+  });
+} else {
+  describe('we do not want this test to run locally, so we mock out a test to make it skip', () => {
```

Comment:
Not a deal breaker: Else statement worth it ?

Reply:
I think it is!